acoustid/data/track.py

```
@@ -79,15 +79,19 @@
-    query = sql.select(
-        [schema.track_mbid.c.track_id, schema.track.c.gid, schema.track_mbid.c.mbid],
-        sql.and_(
-            schema.track_mbid.c.mbid.in_(mbids), schema.track_mbid.c.disabled.is_(False)
-        ),
-        from_obj=schema.track_mbid.join(
-            schema.track, schema.track_mbid.c.track_id == schema.track.c.id
-        ),
-    ).order_by(schema.track_mbid.c.track_id)
+    query = (
+        sql.select(
+            schema.track_mbid.c.track_id, schema.track.c.gid, schema.track_mbid.c.mbid
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)